pkg/spec/cluster_tls.go

```diff
@@ -22,8 +22,9 @@ type TLSPolicy struct {
 }
 
 type StaticTLS struct {
-	// ServerSecretName contains peer-interface and client-interface server x509 key/cert, along with peer and client CA cert.
+	// ServerSecretName contains peer-interface and client-interface server x509 key/cert and CA cert.
```

Comment:
First, separating them to make it clearer sounds good.

> have a cluster with only secured peer communication or only secured client communication.

I think we can start with requiring both to be secured first.
We can have such functionality to turn on either optionally later.